### PR TITLE
Add an upload Component to the File Upload Fields Section

### PR DESCRIPTION
**Pull request by AmirShelli**
This pull request introduces the `Upload` component to the UI samples' File Upload Fields section.
Added their description and its Russian localization.

### CODE REVIEW

File: src/main/java/io/jmix/uisamples/view/flowui/components/fileuploadfields/upload/uploadSample.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import io.jmix.flowui.kit.component.upload.event.FileUploadSucceededEvent;
+import io.jmix.flowui.view.StandardView;
+import io.jmix.flowui.view.Subscribe;
+import io.jmix.flowui.view.ViewController;
+import io.jmix.flowui.view.ViewDescriptor;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@ViewController("upload")
+@ViewDescriptor("upload.xml")
+public class uploadSample extends StandardView {
```
Comment (honest-niceman) [inline src/main/java/io/jmix/uisamples/view/flowui/components/fileuploadfields/upload/uploadSample.java:17]: in camelCase? All java classes should be named in CamelCase

**Review by honest-niceman — CHANGES_REQUESTED** [review]
One change required

**Review by honest-niceman — APPROVED** [review]

**Comment by honest-niceman** [comment]
@knstvk please take a look, lgtm

**Comment by knstvk** [comment]
👍

**Pull request merged**